docs/modules/getting_started/examples/code/getting_started.sh.j2

```diff
@@ -0,0 +1,103 @@
+fi
+
+echo "Adding bitnami Helm Chart repository and dependencies (Postgresql and Redis)"
+# tag::helm-add-bitnami-repo[]
+helm repo add bitnami https://charts.bitnami.com/bitnami
+# end::helm-add-bitnami-repo[]
+# tag::helm-add-bitnami-pgs[]
+helm install --wait airflow-postgresql bitnami/postgresql --version {{ versions.postgresql }} \
+    --set auth.username=airflow \
+    --set auth.password=airflow \
+    --set auth.database=airflow
+# end::helm-add-bitnami-pgs[]
+# tag::helm-add-bitnami-redis[]
+helm install --wait airflow-redis bitnami/redis \
+    --set auth.password=redis \
+    --version {{ versions.redis }} \
+    --set replica.replicaCount=1
+# end::helm-add-bitnami-redis[]
+
+case "$1" in
+"helm")
+echo "Adding 'stackable-dev' Helm Chart repository"
+# tag::helm-add-repo[]
+helm repo add {{ helm.repo_name }} {{ helm.repo_url }}
+# end::helm-add-repo[]
+echo "Installing Operators with Helm"
+# tag::helm-install-operators[]
+helm install --wait commons-operator {{ helm.repo_name }}/commons-operator --version {{ versions.commons }}
+helm install --wait secret-operator {{ helm.repo_name }}/secret-operator --version {{ versions.secret }}
+helm install --wait airflow-operator {{ helm.repo_name }}/airflow-operator --version {{ versions.airflow }}
+# end::helm-install-operators[]
+;;
+"stackablectl")
+echo "installing Operators with stackablectl"
+# tag::stackablectl-install-operators[]
+stackablectl operator install \
+  commons={{ versions.commons }} \
+  secret={{ versions.secret }} \
+  airflow={{ versions.airflow }}
+# end::stackablectl-install-operators[]
+;;
+*)
+echo "Need to give 'helm' or 'stackablectl' as an argument for which installation method to use!"
+exit 1
+;;
+esac
+
+echo "Creating credentials secret"
+# tag::apply-airflow-credentials[]
+kubectl apply -f airflow-credentials.yaml
+# end::apply-airflow-credentials[]
+
+echo "Creating Airflow cluster"
+# tag::install-airflow[]
+kubectl apply -f airflow.yaml
+# end::install-airflow[]
+
+echo "Waiting on AirflowDB ..."
+# tag::wait-airflowdb[]
+kubectl wait airflowdb/airflow \
+  --for jsonpath='{.status.condition}'=Ready \
+  --timeout 300s
+# end::wait-airflowdb[]
+
+sleep 5
+
+echo "Awaiting Airflow rollout finish"
+# tag::watch-airflow-rollout[]
+kubectl rollout status --watch statefulset/airflow-webserver-default
+kubectl rollout status --watch statefulset/airflow-worker-default
+kubectl rollout status --watch statefulset/airflow-scheduler-default
+# end::watch-airflow-rollout[]
+
+echo "Starting port-forwarding of port 8080"
+# tag::port-forwarding[]
+kubectl port-forward svc/airflow-webserver 8080 2>&1 >/dev/null &
+# end::port-forwarding[]
+PORT_FORWARD_PID=$!
+trap "kill $PORT_FORWARD_PID" EXIT
+sleep 5
+
+echo "Checking if web interface is reachable ..."
+return_code=$(curl -s -o /dev/null -w "%{http_code}" http://localhost:8080/login/)
+if [ "$return_code" == 200 ]; then
+  echo "Webserver UI reachable!"
+else
+  echo "Could not reach Webserver UI."
+  exit 1
+fi
```

Comment:
nice! But is it possible to also script the next parts that are going on in the UI? I mean triggering a DAG and checking if it ran through. I did that for druid: https://docs.stackable.tech/druid/nightly/getting_started/first_steps.html#_ingest_example_data

Reply:
Good idea - I've added a health check and DAG run-and-check at the end.